server/app.js

```diff
@@ -240,158 +151,73 @@ function stop_state_machine(){
 }
 
 function call_video_stream() {
-    console.log("Calling video stream script.\n");
+    debug_log("Calling video stream script.\n");
     var pyshell = new PythonShell('../sidewalk/python/video_stream.py')
 }
 
 function navigating_sidewalk(){
     //Read
-    current_state = state[0];
+    current_state = state["NAVIGATING_SIDEWALK"];
 
-    console.log("navigating sidewalk");
+    debug_log("navigating sidewalk");
 
     var file = 'front.json';
     jsonfile.readFile(file, function(err, obj) {
         //console.log(obj.angle);
 
         if(obj.angle >= 85 && obj.angle <= 95 ){
-                client.connect(5000, "192.168.43.250", function() { //192.168.43.250 port 5000
-                    console.log('Connected to 192.168.43.250'); 
-                    console.log('Local client sending:', 0);
-                    buf = new Buffer([0, 0x1E, 0xFF,18, 0, 0xFF]); // 0x4B is temp value 75
-                    client.write(buf);
-
-                    client.destroy();
-                    client = new net.Socket();
-                    client.on('err', function(err){
-                        // handle the error safely
-                        console.log(err)
-                    });
-                });        
+            clientConnectSendState(0,0x1E,0);       
```

Comment:
Can these variables be named?
- What does 0 mean?
- What does 0x1E mean?
- What does 0 mean?

Reply:
Ditto for the other related function calls.